screens/visualization/sketch.py

```diff
@@ -104,16 +103,32 @@ def __call__(self):
         return results
 
 
-if __name__ == '__main__':

```

Comment:
I think we should keep this but further to the end, so that
```
run -i screens/visualization/sketch.py
```
from an ipython session continues to work (and ideally we ensure that `python3 <same>` keeps the display up rather than drawing it and then cutting out!).

That way the routine is still able to reproduce the figures I wrote it for (which are then a nice example of how to use the function).

So, the idea would be to move just the essentials to the function.

Reply:
I think I got most of this to work. However, running `python3 -i screens/visualization/sketch.py` opens an additional unnecessary figure window for me.

Reply:
No unnecessary and/or empty figure windows anymore for me.